src/Panel/Panel.jsx

```diff
@@ -0,0 +1,48 @@
+import _ from "lodash";
+import classnames from "classnames";
+import React, {PropTypes} from "react";
+
+import "./Panel.less";
+
+/*
+ * Panel is various types of panel.
+*/
+export default function Panel({children, type, title}) {
+  const {Icons, cssClass} = Panel;
+
+  return (
+    <div className={classnames(`Panel--${type}`, cssClass.CONTAINER)}>
+      <div className={cssClass.HEADER}>
+        <i className={classnames("fa", Icons[type])} />
```

Comment:
Does nesting not make sense here? The nested syling only makes sense/is applied when inside the relevant box? Happy to change if we want to avoid all nesting but I feel it is easier to understand like this.